apps/web/src/components/editor/index.tsx

```diff
@@ -559,6 +560,23 @@ export function Editor(props: EditorProps) {
           const link = await NoteLinkingDialog.show({ attributes });
           return link || undefined;
         }}
+        onAutoSaveDisabled={() => {
+          setEditorSaveState(id, SaveState.NotSaved);
+          const { hide } = showToast(
+            "error",
+            "Auto-save is disabled for large notes. Press Ctrl + S to save.",
+            [
+              {
+                text: "Dismiss",
+                onClick: () => {
+                  hide();
```

Comment:
The auto save toast appears again on change after dismissing it once. We should store it in memory (not persistent) if user dismisses it.

Secondly, switching notes doesn't get rid of the toast.